contentScript.js

```diff
@@ -250,7 +252,7 @@ setInterval(function () {
         $.ajax({
           url: "https://api.countapi.xyz/hit/cowinbooking/booked4",
         });
-        // play apointment confirm sound here
+        confirmAudio.play().catch(_ => {});
```

Comment:
You have to add the alert in the catch block!

Reply:
He had added. I removed it. 
IMO, if the user is going to see the alert on the screen in case audio doesn't play (unlikely), he might as well see the actual page instead of an alert!

Reply:
It was removed in this [commit](https://github.com/sushrut111/cowin-automation-extn/pull/72/commits/76f55803fd11136fd1ff15d103377968729b8aaf). Initially, I had the alert for the same.

Reply:
Oh okay, cool. I thought it was missed. Resolved

Reply:
@prayuj if that makes sense, please approve and merge it.